### DISCUSSION TITLE
Is it possible to use fzf when completing command parameters?

### DISCUSSION THREAD

**Discussion by ianliu** [junegunn/fzf#2756] [Q&A]
I would like, for example, to use fzf to show me the completion options for bash-completion. When using `awscli`, which has loads of options, it would be very nice to explore the arguments with fzf. Is this possible?

**Comment by DevAtDawn** [comment]
@ianliu did this for fish shell, maybe check out fzf-tab for zsh:
https://github.com/DevAtDawn/fish-fzf-tab
```
function clia
    read -l line; commandline -a $line
    # commandline -a 
end

bind -M insert \et 'fuzzy_complete'

function fuzzy_complete
    complete -C | sort -u | fzf --height 40% --multi --reverse -q (commandline -t) | cut --output-delimiter ' ' -f1 | sed 's/-//g' | clia
    commandline -f end-of-line 
end
```
